src/js/UACustomEvent.js

```diff
@@ -39,9 +43,9 @@ class UACustomEvent {
      * Adds a property to the custom event.
      *
      * @param {string} name The property name.
-     * @param {string|number|boolean|string[]} value The property value.
+     * @param {string|number|boolean|JsonValue} value The property value.
```

Comment:
if you're going to use JsonValue you don't need the other types in this union, as it's inclusive of them